build.ps1

```diff
@@ -198,17 +199,27 @@ function Build-CMakeProject
 
     $RuntimeBuildDir = Get-ProjectBuildDir $Arch 1
     $SwiftResourceDir = "${RuntimeBuildDir}\lib\swift"
-    $SwiftcFlags = @(
-      "-resource-dir $SwiftResourceDir",
-      "-L $SwiftResourceDir\windows",
-      "-vfsoverlay $RuntimeBuildDir\stdlib\windows-vfs-overlay.yaml",
-      "-g -debug-info-format=codeview",
-      "-Xlinker /INCREMENTAL:NO",
-      "-Xlinker /DEBUG",
-      "-Xlinker /OPT:REF",
-      "-Xlinker /OPT:ICF"
-    ) -Join " "
 
+    $SwiftArgs = [System.Collections.ArrayList]@()
```

Comment:
Why an explicit ArrayList?

Reply:
The `@()` is fixed in size and causes an exception.  Am I doing something wrong?

Reply:
When using `.Add`? That method shouldn't be part of the array interface but maybe it would be with Powershell. I'm using `+` or `+=` to concatenate arrays successfully elsewhere in this script (maybe it's reallocating but I don't think we care)

Reply:
Yes, the `+=` reallocates